types/wlr_output_management_v1.c

```diff
@@ -240,7 +240,7 @@ static void config_head_handle_set_scale(struct wl_client *client,
 		return;
 	}
 
-	double scale =  wl_fixed_to_double(scale_fixed);
+	float scale =  wl_fixed_to_double(scale_fixed);
```

Comment:
This makes me feel icky.  We could also go the route of making `wlr_output.scale` a double instead of a float, but that would be a more widely reaching change.